package.json

```diff
@@ -31,25 +38,28 @@
     "url": "git+https://github.com/MichalZalecki/react-portal-universal.git"
   },
   "author": "Michal Zalecki <michal@michalzalecki.com>",
+  "contributors": [
+    "Jess Telford <hi+npm@jes.st> (https://jes.st)"
```

Comment:
Feel free to add the "Contributors" section to the README. I doubt many people will see this in the package.json file.